tests/include/tc_util.h

```diff
@@ -12,7 +12,7 @@
 #include <zephyr.h>
 
 #include <string.h>
-#include <shell/legacy_shell.h>
+#include <shell/shell.h>
```

Comment:
All unit tests are failing because of this..
```
In file included from include/logging/log.h:11:0,
                 include/shell/shell.h:16,
                 tests/include/tc_util.h:15,
                 tests/ztest/include/ztest.h:62,
                 tests/ztest/src/ztest.c:7:
include/logging/log_core.h:19:2: error: #error "Logger does not support 64 bit architecture."
 #error "Logger does not support 64 bit architecture."
```
Unit tests are compiled fully natively (there is no posix arch involved)